docs/narr/i18n.rst

```diff
@@ -429,7 +428,7 @@ Performing a Translation
 
 A :term:`localizer` has a ``translate`` method which accepts either a
 :term:`translation string` or a Unicode string and which returns a Unicode
-object representing the translation.  Generating a translation in a view
+string representing the translation.  Generating a translation in a view
```

Comment:
Should the preceding "Unicode" be removed here?

Reply:
No, I chose to keep it that way. It doesn't hurt to mention that it's a string supporting unicode codepoints, versus a string supporting only ascii or latin-1 codepoints. I'm actually planning to go through some of my earlier edits and possible bring back the word Unicode in a few places.